source/workers/smelter.md

```diff
@@ -34,3 +34,7 @@ Smelters have a chance to double and even triple ores, depending on the Smeltery
 | 5 | --- | 100% | 2x Strength Level |
 
 The higher a Smelter's Athletics level, the faster their furnaces will smelt.
+
+---
+
+In 1.18, the Smelter instead applies Fortune to the ores they smelt - the level of Fortune is 1 less than the level of the smelter.  See the [Minecraft wiki](https://minecraft.fandom.com/wiki/Fortune#Ore) for details on Fortune.  Fortune IV (not on the table) will average 2.67 ores per ore.  This bonus doesn't apply to raw iron, copper, and gold chunks that the colony is given by the player, as it only applies to ores. 
```

Comment:
Can you split the first sentence into two at the en-dash? Grammatically, it doesn't really make sense for a en- or an em-dash to be there.

Reply:
Oh! Turns out, I can directly edit it. Never mind on this, now!